share/plugins/notenames.qml

```diff
@@ -18,16 +18,19 @@ import QtQuick 2.0
 import MuseScore 3.0
 
 MuseScore {
-   version: "3.0"
+   version: "3.3"
    description: qsTr("This plugin names notes as per your language setting")
-   menuPath: "Plugins.Notes." + qsTr("Note Names") // this does not work, why?
+   menuPath: "Plugins.Notes." + qsTr("Note Names")
 
-   function nameChord (notes, text) {
+   // Small note name size is fraction of the full font size.
+   property var fontSizeMini: 0.7;
+
+   function nameChord (notes, text, fontSize) {
       for (var i = 0; i < notes.length; i++) {
-         var sep = ","; // change to "\n" if you want them vertically
+         var sep = "\n";   // change to "," if you want them horizontally (anybody?)
```

Comment:
@Jojo-Schmitz, that seems to be your code originally, do you agree with changing this default?

Reply:
Please check out my examples at the issue web page to see why I made that switch.

-Dale

Reply:
@dmitrio95 I'm fine with that, and it isn't really my code alone anyway